docs/topics/installing-web-console-or-cli-tool.adoc

```diff
@@ -51,13 +51,13 @@ ifdef::web-console-guide[]
 * Linux operating system:
 +
 ----
-$ <MTA_HOME>/run_mta.sh
+$ <{ProductShortName}_HOME>/run_mta.sh
```

Comment:
This renders as <MTR_HOME>/run.mta.sh  
\. Is that the script name? I would guess it should be <MTR_HOME>/run.mtr.sh  or in asciidoc, <{ProductShortName}_HOME>/run_.sh{LC_PSN}